.cirrus.yml

```
@@ -37,4 +36,9 @@ task:
     # Bundle and upload sources
     - sentry-cli debug-files bundle-sources softnet.dSYM/
     - sentry-cli debug-files upload -o $SENTRY_ORG -p $SENTRY_PROJECT softnet.src.zip
-  release_script: goreleaser
+  create_sentry_release_script:
+    - export SENTRY_RELEASE="softnet@$CIRRUS_TAG"
+    - sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
```

I don't think this needs the `-p` argument since we already have the `SENTRY_PROJECT` variable?

```suggestion
    - sentry-cli releases new $SENTRY_RELEASE
```